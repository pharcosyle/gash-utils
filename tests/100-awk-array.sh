echo -e "foo\nbar" | awk -f tests/data/awk/array.awk | LANG=C LC_ALL=C sort
