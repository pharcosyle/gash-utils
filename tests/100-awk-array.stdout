bar => 1
baz => 
foo => 0
have foo
